app/scripts/modules/core/src/pipeline/config/triggers/artifacts/helm/HelmArtifactEditor.tsx

```diff
@@ -4,71 +4,98 @@ import { Option } from 'react-select';
 import { ArtifactTypePatterns } from 'core/artifact';
 import { IArtifact, IArtifactEditorProps, IArtifactKindConfig } from 'core/domain';
 import { StageConfigField } from 'core/pipeline';
-import { TetheredSelect } from 'core/presentation';
-
+import { TetheredSelect, TetheredCreatable } from 'core/presentation';
 import { ArtifactService } from '../ArtifactService';
 import { cloneDeep } from 'lodash';
```

Comment:
I know this wasn't you but while you're in here would you mind moving this import up into the same block as the other third-party imports?